src/xrpld/rpc/handlers/AccountObjects.cpp

```diff
@@ -222,7 +222,8 @@ doAccountObjects(RPC::JsonContext& context)
             {jss::xchain_owned_claim_id, ltXCHAIN_OWNED_CLAIM_ID},
             {jss::xchain_owned_create_account_claim_id,
              ltXCHAIN_OWNED_CREATE_ACCOUNT_CLAIM_ID},
-            {jss::bridge, ltBRIDGE}};
+            {jss::bridge, ltBRIDGE},
+            {jss::permissioned_domain, ltPERMISSIONED_DOMAIN}};
```

Comment:
You should also be able to query the ledger entry from the account/seq so add that as well. And then add tests.

Reply:
I added a flag for account_objects to return type=permissioned_domain, and tested it. Also the ledger_entry call is already tested ok. Please specify what other RPC's you think are necessary, or submit a patch if you feel strongly about it. But i think what's there already is sufficient for people to find their PermissionedDomain objects.

Reply:
There are 2 ledger entry calls. The index and `{ permissioned_domain: { account: account, seq: seq }}` the other needs to be added and tested unless I'm just missing it. 

You tested that the objects are empty, you then lower in the test would create the permissioned_domain and validate it. I'm not the main reviewer so if someone else is fine with you testing account_objects differently than the other objects then thats ok. (All the objects are not even tested there anyways so its prob not a big deal)

Reply:
I've checked documentation, it seems we also need to extend the RPCs:
* `ledger_data`- relies on `chooseLedgerEntryType` which is already modified, so we only need to add it to unit tests
* `ledger_entry` - needs a custom section of code inside `doLedgerEntry` (and a unit test)

Reply:
Whenever you add a new type of ledger entry, the following methods need to be extended:

- account_objects - `type` field
- ledger_data - `type` field
- ledger_entry - new field for looking up the new entry type (typically this can accept either a string ledger entry ID or an object with nested sub-fields as appropriate for the type of entry)

I think the `ledger_data` and `account_objects` type filters use shared code as mentioned above.

I don't think there's anything else that always needs to be added/updated, but of course depending on what the amendment does there may always be more. In the case of Permissioned Domains I'm guessing nothing.